### ISSUE TITLE
Multi-label classification

### ISSUE THREAD

**Issue by BovineEnthusiast** [mkocabas/focal-loss-keras#19]
This loss function is for binary focal loss.

**Comment by BovineEnthusiast** [comment]
There is an implementation here:

https://github.com/Umi-you/FocalLoss
